bin/rollup

```diff
@@ -0,0 +1,1566 @@
+#!/usr/bin/env node
```

Comment:
This file was probably checked in accidentally and should be removed. The reason this happened is probably that the bin file has been moved and the .gitignore was changed accordingly.